packages/react/src/backlink/Backlink.tsx

```diff
@@ -0,0 +1,39 @@
+import { forwardRef, type Ref } from 'react';
+import { cx } from 'cva';
+import { Link as RACLink, type LinkProps } from 'react-aria-components';
+import { ArrowLeft } from '@obosbbl/grunnmuren-icons-react';
+
+type BacklinkProps = {
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+
+  /** The URL to navigate to when clicking the breadcrumb. */
+  href?: string;
+
+  /** Icon styling */
+  iconStyle?: string;
+} & Omit<LinkProps, 'className' | 'style'>;
+
+function Backlink(props: BacklinkProps, ref: Ref<HTMLAnchorElement>) {
+  const { className, children, href, iconStyle, ...restProps } = props;
+
+  return (
+    <RACLink
+      className={cx(className, 'flex max-w-fit items-center')}
```

Comment:
Kan vi slenge på litt `py` her for å få en høyde på 44px? [SC 2.5.5 - Target Size](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html)